### PR TITLE
[Swift] Fix the remaining test failures.

### PR DESCRIPTION
**Pull request by dcci**
With this the test suite fully passes.

### CODE REVIEW

**Comment by dcci** [comment]
@swift-ci test and merge

File: packages/Python/lldbsuite/test/lang/swift/playgrounds-repl/old_playground/TestNonREPLPlayground.py [hunk 1]
```diff
@@ -21,9 +20,14 @@
 import os.path
 import unittest2
 
+import sys
+if sys.version_info.major == 2:
+    import commands as subprocess
```
Comment (JDevlieghere) [inline packages/Python/lldbsuite/test/lang/swift/playgrounds-repl/old_playground/TestNonREPLPlayground.py:25]: I'm pretty sure there's a six wrapper for this but no big deal.

**Review by JDevlieghere — APPROVED** [review]

**Comment by dcci** [comment]
@swift-ci test and merge

**Comment by dcci** [comment]
@swift-ci test and merge

**Comment by dcci** [comment]
@swift-ci test and merge

**Comment by dcci** [comment]
@swift-ci test linux

**Pull request merged**